cpp/benchmarks/CMakeLists.txt

```diff
@@ -52,6 +52,26 @@ function(ConfigureBench CMAKE_BENCH_NAME)
     )
 endfunction(ConfigureBench)
 
+# This function takes in a benchmark name and benchmark source for nvbench benchmarks and handles
+# setting all of the associated properties and linking to build the benchmark
+function(ConfigureNVBench CMAKE_BENCH_NAME)
+  add_executable(${CMAKE_BENCH_NAME} ${ARGN})
+  set_target_properties(
+    ${CMAKE_BENCH_NAME}
+    PROPERTIES RUNTIME_OUTPUT_DIRECTORY "$<BUILD_INTERFACE:${CUSPATIAL_BINARY_DIR}/benchmarks>"
+               INSTALL_RPATH "\$ORIGIN/../../../lib"
+  )
+  target_link_libraries(
+    ${CMAKE_BENCH_NAME} PRIVATE cuspatial_benchmark_common nvbench::main
+  )
+  install(
```

Comment:
They both install them, the install command in `ConfigureBench` just doesn't show up in the Github diff unless you expand. At least, I assume that's why you didn't see it, because the same thing happened to me when I reviewed :slightly_smiling_face: 

As for whether or not we _should_ install them, I think this logic is copied from [the corresponding cudf file](https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/benchmarks/CMakeLists.txt#L80). I agree that the installation probably isn't necessary, though.

Reply:
Updated the component name to be `benchmark`.